pkg/api/params/params.go

```diff
@@ -614,10 +615,21 @@ type BootstrapStartParams struct {
 	CloudName string `json:"cloud-name"`
 	// RegionName specifies the target region for the controller.
 	RegionName string `json:"region-name"`
+	// Cloud holds the cloud definition that will be used to bootstrap the controller.
+	Cloud jujuparams.Cloud `json:"cloud,omitempty"`
+	// Credential contains the cloud credential and its tag, this credential will be used against the
+	// the cloud provided to bootstrap the controller.
+	// We're using jujucloud.CloudCredential as there's an API client func using this type to add clouds.
+	// Seen here: api/client/cloud/cloud.go L271.
```

Comment:
Referencing the line number is quite fragile as it will likely change and I'm not sure which branch to look at.